tests/upgrades/test_remoteexecution.py

```diff
@@ -151,70 +160,90 @@ class TestScenarioREXSatellite:
 
         1. Before Satellite upgrade:
         2. Create Content host.
-        3. Create a Subnet on Satellite.
-        4. Install katello-ca on content host.
-        5. Register content host to Satellite.
-        6. Add_ssh_key of Satellite on content host.
-        7. Run a REX job on content host.
-        8. Upgrade satellite/capsule.
-        9. Run a rex Job again with same content host.
-        10. Check if REX job still getting success.
+        3. Register content host to Satellite.
+        4. Run a REX job on content host.
+        5. Upgrade satellite/capsule.
+        6. Run a rex Job again with same content host.
+        7. Check if REX job still getting success.
     """
 
+    @pytest.mark.rhel_ver_list([8])
+    @pytest.mark.no_containers
+    @pytest.mark.skip_checkin
```

Comment:
Why this marker is needed ?

Reply:
Also, I dont see this was added in the last scenario ?

Reply:
let me remove that